LEAF_Request_Portal/Email.php

```diff
@@ -41,16 +41,27 @@ class Email
     private $portal_db;
     private $nexus_db;
 
+    private $siteRoot = "";
+
     public $smartyVariables = array();
 
     const SEND_BACK = -1;
     const NOTIFY_NEXT = -2;
     const NOTIFY_COMPLETE = -3;
+    const EMAIL_REMINDER = -4;
 
     public function __construct()
     {
         $this->initPortalDB();
         $this->initNexusDB();
+
+        $protocol = isset($_SERVER['HTTPS']) && $_SERVER['HTTPS'] == 'on' ? 'https' : 'http';
```

Comment:
Why are we allowing HTTP?

Reply:
This has been changed to remove the protocol check which had been moved over from some older code during the code consolidation.  It is now always set to HTTPS.